src/components/Expensicons/ICON_NAMES.js

```diff
@@ -0,0 +1,9 @@
+export default {
+    CLIPBOARD: 'clipboard',
+    COPY_LINK: 'copyLink',
+    EXPENSIFY_CASH_LOGO: 'expensifyCashLogo',
+    MAIL: 'mail',
+    PENCIL: 'pencil',
+    PIN: 'pin',
+    TRASH: 'trashcan',
+};
```

Comment:
Any thoughts about moving these to the `/styles` directory? I'm thinking they are sort of part of the "design system" and should live there instead of `/components`.